.github/workflows/spelling3.yml

```diff
@@ -24,10 +24,6 @@ jobs:
       contents: read
       # pull-requests-read is needed for suppress_push_for_open_pull_request in private repositories
       pull-requests: read
-      # actions-read is needed (possibly only for private repositories)
-      # to identify the workflow's filename until
-      # https://github.com/actions/runner/issues/853 is fixed
-      actions: read
```

Comment:
I believe this is no longer necessary for private repositories as I've switched to using a relatively "new" variable which provides the workflow's file name instead...